src/main.js

```diff
@@ -0,0 +1,416 @@
+/* Main
+/ 
+/  run function to setup all the functions
+/  create global variables for fields
+/  setup the connections to fetch data by query and then send to interpret by data.js
+/  load in configuration cookies and selections.
+*/
+
+function debug(message, priority) {
+
+    // Priority debugs are giving an html alert
+    if (priority) {
+        alert(message)
+        console.error(message)
+    } else {
+        console.log(message)
+    }
+}
+
+function initialize() {
+
+    // Render -> render.js
+    Render = new render()
```

Comment:
make variables explicit https://rules.sonarsource.com/javascript/RSPEC-2703/

Reply:
Making variables explicit within a function in javascript has the result that they are no longer globally accessible.
They would need to be placed outside of the initialize function to be explicit. This would mean that main is a bit more depending on being loaded last of all javascript files.